rotkehlchen/icons.py

```diff
@@ -35,6 +35,23 @@
 ALLOWED_ICON_EXTENSIONS = ('.png', '.svg', '.jpeg', '.jpg', '.webp')
 
 
+def _extract_http_type_from_extension(image_path: Path) -> str:
```

Comment:
Why abstract this in a function when it's only used in _build_http_header_for_images? Why not just put it inside there? Both are really small functions and if used in only 1 location there is no reason to do this